presto-druid/src/main/java/com/facebook/presto/druid/DruidPushdownUtils.java

```diff
@@ -150,7 +155,7 @@ private static Number decodeDecimal(BigInteger unscaledValue, DecimalType type)
     }
 
     // Copied from com.facebook.presto.sql.planner.LiteralInterpreter.evaluate
```

Comment:
Maybe we could remove this line or add some more explaination in the future as some changes have been done based on the prior code snippet.